README.rst

```diff
@@ -1,3 +1,7 @@
+.. image:: https://github.com/techservicesillinois/awscli-login/workflows/CI/CD/badge.svg
```

Comment:
I assume that the intervening path separator (slash) in "CI/CD" is not going to cause a broken link because of the browser resolving what looks like an extra path component.

Reply:
It works fine. You can actually try the link and see for yourself: https://github.com/techservicesillinois/awscli-login/workflows/CI/CD/badge.svg.